includes/REST/DokanDataContinentsController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace WeDevs\Dokan\REST;
+
+use WC_REST_Data_Continents_Controller;
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+
+class DokanDataContinentsController extends WC_REST_Data_Continents_Controller {
```

Comment:
- Pls refactor the class just by overriding the `get_items_permissions_check` and try to not override any other methods.
- Must add the test case.